homeassistant/components/climate/__init__.py

```diff
@@ -348,70 +313,6 @@ def add_to_platform_start(
         """Start adding an entity to a platform."""
         super().add_to_platform_start(hass, platform, parallel_updates)
```

Comment:
If you are just calling `super()`, do you need to keep this function?

Reply:
No, I missed that the below function for deprecating `aux` isn't part of this. Has cleaned it off now.